app/views/content/subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -52,9 +52,17 @@
   <section>
     <h2 class="heading-s">Engineers teach physics training programme</h2>
 
-    <p>This new teacher training programme helps engineers to become physics teachers.</p>
+    <p>This new programme is based on the physics initial teacher training (ITT) course. It has been adapted to help you make the transition from engineering or material sciences to teaching physics. The course is designed to capitalise on your experience and empower you with the skills needed to become a great physics teacher.
+     </p>
+
+    <p>Your training provider may offer part-time teacher training courses and may also allow you to combine courses, for example Engineers teach physics combined with mathematics.
```

Comment:
Do you think we maybe want to switch around the part-time courses bit and the SKE bit? Just think it flows slightly better